bundle/Resources/config/doctrine.yaml

```diff
@@ -0,0 +1,10 @@
+orm:
+    entity_managers:
+        default:
+            auto_mapping: false
```

Comment:
`auto_mapping` config here should not be set here as it will overwrite global `auto_mapping` config set in project.

Reply:
@emodric Fixed in https://github.com/netgen-layouts/layouts-remote-media/pull/15/commits/bf5c334bf66ff0cb669d85f62174d003c17912df.